### PR TITLE
Adding a new section for Puppet consulting

### PR DESCRIPTION
**Pull request by glarizza**
This PR is a work in process, and is simply a stub containing the framework
for a new Puppet consulting page. Next steps are deciding on the content
and then spiking it out.

### CODE REVIEW

**Comment by glarizza** [comment]
Did an initial revision on the consulting page.  Here's some screenshots on what that looks like (or you can pull the branch and `rake serve` locally):

Screenshot 2023-01-25 at 11 51 59 AM
Screenshot 2023-01-25 at 11 51 49 AM

File: index.html [hunk 1]
```diff
@@ -33,9 +33,9 @@ <h4 class="service-heading">Cloud Architecture</h4>
           <i class="fa fa-circle fa-stack-2x text-primary"></i>
           <i class="fa fa-code fa-stack-1x fa-inverse"></i>
         </span>
-        <h4 class="service-heading">Puppet Module Development</h4>
+        <h4 class="service-heading">Puppet Consulting</h4>
         <p class="text-muted">
-          We specialize in building Puppet modules to meet your specific needs.
+          We specialize in <a href="./puppet_consulting/">building Puppet modules to meet your specific needs.</a>
```
Comment (glarizza) [inline index.html]: TODO: Revise this section to link up with the new consulting page. This link was added to verify the ability to link to the new Puppet consulting page, and should not be kept without revising the section.

Comment (natemccurdy) [inline index.html]: I think it'd be good, if we do keep this Puppet-related blurb here, to expand it to mention Puppet infrastructure as well as Puppet modules.

For example, Puppet infrastructure scaling, tuning, monitoring, etc...

Comment (natemccurdy) [reply index.html]: Oh, woops... I didn't see that you do call that out in the Puppet-specific page below.

So perhaps then just simplifying the original text of:
> We specialize in building Puppet modules to meet your specific needs.
> Have a custom Java app that needs to be deployed more quickly and reliably?  No problem!  We're here to help you fully automate the  process.

Down into something like:
> We specialize in Puppet infrastructure and module development. Having Puppetserver scaling problems or need a module for your custom app? No problem! We're here to help.


...or something like that.

File: puppet_consulting.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      </div>
+    </div>
+  </div>
+</section>
+
+<section id="facts">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12">
+        <h2 class="section-heading text-center">Fast Facts</h2>
```
Comment (natemccurdy) [inline puppet_consulting.html]: This entire page is nearly perfect. The only thing that stands out is the bullet-point formatting of this "Fast Facts" section.

I wonder if these bullet points would flow better as just a short paragraph of introductory text right below the "We know Puppet" header.

File: index.html (hunk 1 above)
Comment (glarizza) [reply index.html]: Yeah, that link was a proof of concept to figure out how to link to the new page I created. It definitely needs cleaned up once we finalize what's on the consulting page. I'll fix this up once we're happy with the consulting page.

**Comment by glarizza** [comment]
I did some cleanup on the "We know Puppet" paragraph to fix the flow from one point to the next, and added separate `section` declarations for the "We know Puppet" and "Service offering" sections (that way it matches the existing site layout). 

We're getting closer! I'm wondering whether it makes sense to list some of the customers we've worked with previously, or whether we just leave the testimonial(s) at the bottom and call it a day.

**Review by jeffmccune — APPROVED** [review]

**Pull request merged**